packages/admin-api-client/README.md

```diff
@@ -0,0 +1,243 @@
+# @shopify/admin-api-client
+
+<!-- ![Build Status]() -->
+
+[![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](../../LICENSE.md)
+
+<!-- [![npm version](https://badge.fury.io/js/%40shopify%2Fadmin-api-client.svg)](https://badge.fury.io/js/%40shopify%2Fadmin-api-client) -->
+
+The Admin API Client library is for developers who want to interact with Shopify's GraphQL `Admin API`. The features of this library are designed to be lightweight and minimally opinionated.
+
+## Getting Started
+
+To install this package, you can run this in your terminal:
+
+```typescript
+npm install @shopify/admin-api-client -s
+```
+
+## Admin API Client Examples
+
+### Initialize the Admin API Client
+
+```typescript
+import {createAdminApiClient} from '@shopify/admin-api-client';
+
+const client = createAdminApiClient({
+  storeDomain: 'your-shop-name.myshopify.com',
+  apiVersion: '2023-04',
+  accessToken: 'your-admin-api-access-token',
+});
+```
+
+### Query for a product using the client
+
+```typescript
+const operation = `
+  query ProductQuery($id: ID!) {
+    product(id: $id) {
+      id
+      title
+      handle
+    }
+  }
+`;
+
+const {data, errors, extensions} = await client.request(operation, {
+  variables: {
+    id: 'gid://shopify/Product/7608002183224',
+  },
+});
+```
+
+### `createAdminApiClient()` parameters
+
+| Property            | Type             | Description                                                                                                                                                                                                                                          |
```

Comment:
I think the README still needs to be updated regarding the new `retries` and `logger` features.